classes/local/report.php

```diff
@@ -3711,19 +3711,20 @@ public function get_activity_min_and_max_grade($itemid, $grade): string {
      * @param object $instn
      * @return string The self-reflective notes
      */
-    public function get_notes($userid, $gradeitemid, $instn): string {
+    public function get_notes($userid, $gradeitemid, $instn) {
```

Comment:
what does it return if not a string?

Reply:
there was an issue it was returning null, but I have changed the function and should return an empty string now!